tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -1711,10 +1711,12 @@ def _update_request_states(self, scheduled_requests: ScheduledRequests):
             cp_type = cp_config['cp_type']
             if cp_type == CpType.STAR:
                 self._update_request_states_star_attention(scheduled_requests)
+            elif cp_type == CpType.HELIX:
+                # Take the usual route with _update_request_states_tp().
+                pass
             else:
-                assert False, f'Unsupport cp_type {cp_type}'
-        else:
-            self._update_request_states_tp(scheduled_requests)
+                assert False, f'Unsupported cp type {cp_type.name}.'
```

Ruff (0.13.1)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)